Directory.Build.props

```diff
@@ -45,6 +37,19 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <Choose>
```

Comment:
I never did like the Choose, When, Otherwise tags.. they seem so verbose (and span multiple lines because you have to open and close all those tags). imho, it seems more succinct to write it as:

```xml
<PropertyGroup>
   <IsTest>false</IsTest>
   <IsTest Condition="$(MSBuildProjectName.ToLowerInvariant().Contains('test'))">true</IsTest>
<PropertyGroup>
```

Reply:
I originally had that, but I switched to the choose/when because it seemed cleaner :) I'll switch it back